scripts/start.js

```diff
@@ -139,18 +157,92 @@ function openBrowser() {
   opn('http://localhost:3000/');
 }
 
-new WebpackDevServer(compiler, {
+var server = new WebpackDevServer(compiler, {
   historyApiFallback: true,
   hot: true, // Note: only CSS is currently hot reloaded
   publicPath: config.output.publicPath,
   quiet: true
-}).listen(3000, 'localhost', function (err, result) {
+});
+
+server.listen(3000, 'localhost', function (err, result) {
   if (err) {
-    return console.log(err);
+    return vorpal.log(err);
   }
 
   clearConsole();
-  console.log(chalk.cyan('Starting the development server...'));
-  console.log();
+  vorpal.log(chalk.cyan('Starting the development server...'));
+  vorpal.log();
   openBrowser();
+  startCLI();
 });
+
+function startCLI() {
+  vorpal
+    .command('open')
+    .alias('o')
+    .description('Opens your app in browser.')
+    .action(function(args, onFinish) {
+      openBrowser();
+      onFinish();
+    });
+
+  vorpal
+    .command('edit [editor...]')
+    .alias('e')
+    .description(
+      'Opens your app in editor. Defaults to `$EDITOR`; pass an extra string ' +
+      'to specify your editor of choice \n(e.g., `edit sublime text` or `e visual studio code`).'
+    )
+    .action(function(args, onFinish) {
+      var ctx = this;
+
+      var projectRootPath = config.cliInfo.projectRootPath;
+
+      var editor = '';
+      if (args.editor) {
+        editor = args.editor.join(' ');
+      } else if (process.env.EDITOR) {
+        editor = process.env.EDITOR;
+      } else {
+        ctx.log(
+          'Oops, no editor was found. You can pass an extra string ' +
+          'to specify your editor of choice (e.g., `edit sublime text` or `e visual studio code`).'
+        );
+        onFinish();
+      }
+
+      ctx.log(chalk.blue(
+        'Opening `' + projectRootPath + '` in `' + editor + '`...\n'
+      ));
+
+      opn(projectRootPath, {app: editor}).catch(function(e) {
+        ctx.log(
+          'Oops, we tried to open your app with `' + editor + '` but it didn\'t ' +
+          'work. Please check if `' + editor + '` is an application. ' +
+          'You can pass a second argument to specify ' +
+          'your editor of choice (e.g., `edit sublime text` or `e visual studio code`).'
+        );
+      })
+      onFinish();
+    });
+
+  vorpal
+    .command('verbose')
+    .alias('v')
+    .description('Verbose mode (show all webpack logs).')
+    .action(function(args, onFinish) {
+      isInVerboseMode = !isInVerboseMode;
+      this.log(chalk.blue(
+        'Verbose mode ' + (!isInVerboseMode ?
+          'off; will only show webpack errors.' :
+          'on; will show all webpack logs.')
+      ));
+      onFinish();
+    });
+
+  vorpal.history('create-react-app-cli');
+
+  vorpal
```

Comment:
We can use `vorpal.parse(process.argv)` to jump straight into a command [if extra args are passed into `npm start`](http://stackoverflow.com/questions/11580961/sending-command-line-arguments-to-npm-script).

[Reference on `vorpal.parse`.](https://github.com/dthree/vorpal/wiki/API-%7C-vorpal#vorpalparseargv-options)